cmd/kubeadm/app/constants/constants.go

```diff
@@ -138,6 +139,9 @@ var (
 	// DefaultTokenUsages specifies the default functions a token will get
 	DefaultTokenUsages = []string{"signing", "authentication"}
 
+	// DefaultTokenGroups specifies the default groups a token will get
+	DefaultTokenGroups = []string{bootstrapapi.BootstrapDefaultGroup}
```

Comment:
Isn't it: the less deps the rest of the kubernetes/kubernetes code has on `cmd/kubeadm`; the easier it will be to split out? We already vendor `bootstrapapi` quite naturally, as we're the only user of it currently (AFAIK)